tools/src/tester/TestResults.ts

```diff
@@ -45,13 +46,30 @@ export default class TestResults {
     return this._unevaluated_operations
   }
 
+  ignorable_operations(): Operation[] {
+    if (this._ignorable_operations !== undefined) return this._ignorable_operations
+
+    this._ignorable_operations = _.uniqWith(_.compact(Object.entries(this._spec.spec().paths).flatMap(([path, ops]) => {
+      return Object.entries(ops as Record<string, any>).map(([method, spec]) => {
+        if (spec['x-ignorable'] === true) {
+          return { method: method.toUpperCase(), path }
+        }
+      })
+    })), isEqual)
+
+    return this._ignorable_operations
+  }
```

Comment:
Where is this method used?

Reply:
Was just leftoever from some older version, deleted.